flexget/plugins/filter/list_match.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginError
+
+log = logging.getLogger('list_match')
+
+
+class ListMatch(object):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'from': {'type': 'array',
+                     'items':
+                         {'allOf': [
+                             {'$ref': '/schema/plugins?group=list'},
+                             {
+                                 'maxProperties': 1,
+                                 'error_maxProperties': 'Plugin options within list_match plugin must be indented '
+                                                        '2 more spaces than the first letter of the plugin name.',
+                                 'minProperties': 1
+                             }
+                         ]
+                         }
+                     },
+            'action': {'type': 'string', 'enum': ['accept', 'reject'], 'default': 'accept'},
+            'remove_on_match': {'type': 'boolean', 'default': True},
+            'single_match': {'type': 'boolean', 'default': False},
```

Comment:
hmm, you think? im fine either way really. i figured wanting only one result is something that should explicitly be stated, but i dont have a strong opinion on this.